sync/sync.go

```diff
@@ -241,11 +242,11 @@ func (sync *synchronizer) sendBlocksRequestIfWeAreBehind() {
 			if err := sync.network.JoinDownloadTopic(); err != nil {
 				sync.logger.Info("We can't join download topic", "err", err)
 			} else {
-				sync.RequestForMoreBlock()
+				sync.downloadBlocks()
 			}
 		} else {
 			sync.logger.Info("We are behind the network, Ask for more blocks")
```

Comment:
`behind the network` feel like supporting network I'd suggest something like `less than network height ` or `shorter than network`

Reply:
Thanks. Done.